src/manager/agents/common-module/module/utils/common_utils.cpp

```diff
@@ -105,6 +109,21 @@ bool CommonUtils::TransformToScStruct(
         parameterValueAddr = context.CreateNode(ScType::LinkConst);
         context.SetLinkContent(parameterValueAddr, parameterValue);
       }
+      else if (parameter.first == CommandsConstantsFlags::NOTE)
+      {
+        parameterValueAddr = context.CreateNode(ScType::LinkConst);
+        context.SetLinkContent(parameterValueAddr, parameterValue);
+      }
+      else if (parameter.first == CommandsConstantsFlags::PURPOSE)
+      {
+        parameterValueAddr = context.CreateNode(ScType::LinkConst);
+        context.SetLinkContent(parameterValueAddr, parameterValue);
+      }
+      else if (parameter.first == CommandsConstantsFlags::ID)
+      {
+        parameterValueAddr = context.CreateNode(ScType::LinkConst);
+        context.SetLinkContent(parameterValueAddr, parameterValue);
+      }
```

Comment:
easier not to repeat equal code
```suggestion
      else if (parameter.first == CommandsConstantsFlags::NOTE || ...) 
      {
        parameterValueAddr = context.CreateNode(ScType::LinkConst);
        context.SetLinkContent(parameterValueAddr, parameterValue);
      }
```
or create the list of flagsAddrs and check, if parameter.first is in this list